Formula/kafka.rb

```diff
@@ -94,14 +94,15 @@ def plist
 
       sleep 30
 
-      system "#{bin}/kafka-topics --zookeeper localhost:2181 --create --if-not-exists --replication-factor 1 " \
-             "--partitions 1 --topic test > #{testpath}/kafka/demo.out 2>/dev/null"
-      pipe_output "#{bin}/kafka-console-producer --broker-list localhost:9092 --topic test 2>/dev/null",
-                  "test message"
-      system "#{bin}/kafka-console-consumer --bootstrap-server localhost:9092 --topic test --from-beginning " \
-             "--max-messages 1 >> #{testpath}/kafka/demo.out 2>/dev/null"
-      system "#{bin}/kafka-topics --zookeeper localhost:2181 --delete --topic test >> #{testpath}/kafka/demo.out " \
+      system "#{bin}/kafka-topics --bootstrap-server localhost:9092 --create --if-not-exists " \
```

Comment:
can this use something like 
```suggestion
      port = free_port
      system "#{bin}/kafka-topics --bootstrap-server localhost:#{port} --create --if-not-exists " \
```

Reply:
I'm not sure what the policy is for homebrew-core but Kafka has these well known ports and if we change for `kafka-topics` we'll need to change the server port(s) as well.

Reply:
There's no such policy (that I know of), but it just makes sense to run zookeeper-server and kafka-server on known-free ports (determined with free_port), and configure all the test clients to connect to those ports. That ensures that all these tests run against the daemons you just started, rather than old Kafka servers that happen to be running on the system...or some non-Kafka servers that are squatting on the same ports.

Reply:
Please review `free_port` usage update.